lib/matplotlib/tests/test_backend_qt5.py

```diff
@@ -133,7 +135,7 @@ def test_control_alt():
 def test_modifier_order():
     assert_correct_key(QtCore.Qt.Key_Aacute,
                        (ControlModifier | AltModifier | SuperModifier),
-                       'ctrl+alt+super+' + unichr(225))
+                       'ctrl+alt+super+' + chr(225))
```

Comment:
Maybe this should be six.unichr so it doesn't fail if the default encoding is something that can't handle Unicode codepoint 225...